tuning: recommend external authentication (#1288)
rancher/rancher-docs: docs/reference-guides/best-practices/rancher-server/tuning-and-best-practices-for-rancher-at-scale.md

* Kubernetes permissions are always "additive" (allow-list) rather than "subtractive" (deny-list). Try to minimize configurations that gives access to all but one aspect of a cluster, project, or namespace, as that will result in the creation of a high number of `RoleBinding` objects.
* Experiment to see if creating new projects or clusters manifests in fewer `RoleBindings` for your specific use case.

> #### Use external authentication

Reviewer: ```suggestion
### Using External Authentication
```

> For any use case with more than 50 users configuring an [external authentication](../../../how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/authentication-config/authentication-config.md) provider with support of groups is necessary to attain good performance.

Reviewer: ```suggestion
If you have fifty or more users, you should configure an [external authentication provider](../../../how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/authentication-config/authentication-config.md). This is necessary for better performance.
```

> After configuring external authentication, make sure permissions are assigned to groups instead of individual users, as that will reduce the count of `RoleBinding` objects that Rancher will create.

Reviewer: ```suggestion
After you configure external authentication, make sure to assign permissions to groups instead of to individual users.  This helps reduce the `RoleBinding` object count.
```
